pkg/apis/nmstate/v1alpha1/nodenetworkconfigurationpolicy_types.go

```diff
@@ -60,6 +60,7 @@ var NodeNetworkConfigurationPolicyConditionTypes = [...]ConditionType{
 }
 
 const (
+	NodeNetworkConfigurationPolicyConditionUnknownState                ConditionReason = "UnknownState"
```

Comment:
The value should be a reason (`ConditionReason`). So in this case, maybe `"DesiredStateHasChanged"`?

Reply:
Not needed, also it could be that desiredState is not changed but we re-apply it.